### ISSUE TITLE
Stuck on reading chunk TXTR

### ISSUE THREAD

**Issue by Dolphinz1** [UnderminersTeam/UndertaleModTool#1802]
### Describe the bug

Hello, im very new to this modding tool, and it was working great until this popped up. Now my dat.win file won't open up and its stuck on this. Any way to fix this or is all my work lost?
![Screenshot 2024-06-07 011214](https://github.com/UnderminersTeam/UndertaleModTool/assets/172007857/e36dcf9a-b723-434d-8d48-522f07c3c7c3)

### Reproducing steps

1. Open data.win
2. Stuck on this.

### Setup Details

Im using the newest version on Windows 11

**Comment by Jacky720** [comment]
What game have you been modding, what did you add (especially most recently), and could you make a patch to send over for analysis?

**Comment by Dolphinz1** [comment]
> What game have you been modding, what did you add (especially most recently), and could you make a patch to send over for analysis?

I was modding normal undertale, and all I added was script for special dialogue when you put specific names, and when you say patch do you mean sending my data file over? Sorry I’m new to this.

**Comment by Dolphinz1** [comment]
> A patch would typically be a .xdelta file generated using a tool like DeltaPatcher to let people with the original game recreate your data file. It's a smaller upload and avoids containing 90% of the game data (which we don't like for legal reasons).

alright here, this is it, i tested that the mod tool lets me mod other datas but the one i had progress on just won't open. so heres the file for that now.

**Comment by Dolphinz1** [comment]
> A patch would typically be a .xdelta file generated using a tool like DeltaPatcher to let people with the original game recreate your data file. It's a smaller upload and avoids containing 90% of the game data (which we don't like for legal reasons).

Yes i have the patch, how ever github isn't letting me send it through this? 
how would i get the file to you?

**Comment by Jacky720** [comment]
My DMs (space_core_0352) are open in the Underminers Discord: [https://discord.gg/3ESNF4QPrh](https://discord.gg/3ESNF4QPrh)

**Comment by Jacky720** [comment]
After analysis (GitHub doesn't accept xdelta but it does take zip),
[twttale_patch.zip](https://github.com/user-attachments/files/15741945/twttale_patch.zip)
it seems the file just stopped writing at 0x1000000 (16 MB).

**Comment by Miepee** [comment]
That sadly isnt really actionable :/

**Comment by Miepee** [comment]
We should probably handle those kinds of cases better, the GUI should just flat out error out at that point instead of infinitely looping.
